src/main/resources/db/migration/V8__material_aid_offer.sql

```diff
@@ -5,7 +5,5 @@ create table material_aid_offer (
     description     text,
     category        text,
     city            text,
-    region          text,
-
-    CONSTRAINT PK_MAO_ID PRIMARY KEY (ID)
```

Comment:
Wasn't the primary key a good thing? Removed by accident?

Reply:
wasn't sure if it should be here, other tables has no such declarations

Reply:
All of them should have primary keys. But that's maybe a separate task to review that along with indexes